src/channel.js

```diff
@@ -0,0 +1 @@
+module.exports = '__THEMING__';
```

Comment:
It works until it doesn't, it is so easy to fix without any side effects, why not to?